javascript/extractor/lib/typescript/src/common.ts

```diff
@@ -1,20 +1,43 @@
 import * as ts from "./typescript";
 import { TypeTable } from "./type_table";
+import * as pathlib from "path";
+import { VirtualSourceRoot } from "./virtual_source_root";
+
+/**
+ * Extracts the package name from the prefix of an import string.
+ */
+const packageNameRex = /^(?:@[\w.-]+[/\\])?\w[\w.-]*(?=[/\\]|$)/;
```

Comment:
Can import paths contain double slashes?

Reply:
Indeed that can and it seems TypeScript will happily resolve them. We should handle it now.